game-app/game-core/src/main/java/games/strategy/triplea/ui/EditPanel.java

```diff
@@ -481,7 +485,7 @@ public void actionPerformed(final ActionEvent event) {
           }
         };
     changePUsAction =
-        new AbstractAction("Change PUs") {
+        new AbstractAction("Change PUs/Resources") {
```

Comment:
How do you feel about the "/" ?

IMO this would be better if it were just "Change Resources".
We could alternatively try to compute if there is only one resource and use that for the name. That seems complex and arguably inconsistent. What are your thoughts to simply changing the button title to "Change Resources?"

Reply:
I don't have a strong opinion, so changing seems fine. The only reason is discoverability - someone may not realize that PUs are a type of resources. But this is edit mode, which is a pretty advanced feature to begin with, so I'm not too worried.